osu.Game.Rulesets.Osu/Difficulty/Skills/OsuStrainSkill.cs

```diff
@@ -28,6 +28,8 @@ public abstract class OsuStrainSkill : StrainSkill
         /// </summary>
         protected virtual double DifficultyMultiplier => 1.06;
 
+        protected override int HistoryLength => 10; // Look back for 10 notes is added for the sake of flashlight calculations.
+
```

Comment:
Are you sure you want this done at the base `OsuStrainSkill`, rather than just the `Flashlight` one?

Reply:
Good idea, I will change that